src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin-top: 20px;
+}
+
+.big-form {
+  margin-bottom: 20px;
+}
+
+.big-search {
+  outline: none;
+  width: 100%;
+  height: 70px;
+  background-image: url(images/Search.svg);
+  background-size: 19px;
+  background-repeat: no-repeat;
+  background-position: left 26px top 25px;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
move it to the `html`

Reply:
I do not quite understand .. Prescribe a font for the tag html. And with the background what to do? Prescribe it as a image src and set a class?

Reply:
it's better to move fonts to the HTML selector. Other styles should stay here

Reply:
If I choose a html selector and set a font, then do not pass the tests in the VScode.But if the font is given for the . search, then everything is ok.